### PR TITLE
[COMCTL32_WINETEST] treeview.c: Skip test_right_click(), until CORE-14975 is fixed

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

JIRA issue: [CORE-14975](https://jira.reactos.org/browse/CORE-14975)

### CODE REVIEW

File: modules/rostests/winetests/comctl32/treeview.c [hunk 1]
```diff
@@ -2724,6 +2724,11 @@ static void test_right_click(void)
     LRESULT result;
     POINT pt;
 
+#ifdef __REACTOS__
+    win_skip("test_right_click(), until CORE-14975 is fixed upstream and WINESYNC\n");
```
Comment (ThFabba) [inline modules/rostests/winetests/comctl32/treeview.c]: Guard this by `!winetest_interactive` please

**Comment by ColinFinck** [comment]
> I don't see the point

It would allow Test_VBox to complete again. On the other hand, if we hide the actual problem, it will hardly get fixed..

File: modules/rostests/winetests/comctl32/treeview.c (hunk 1 above)
Comment (SergeGautherie) [reply modules/rostests/winetests/comctl32/treeview.c]: Thanks for the reminder! Done.

**Comment by SergeGautherie** [comment]
bq. allow Test_VBox to complete again

Exactly: unhide whatever may happen after this test (failure).

bq. if we hide the actual problem, it will hardly get fixed.

A limited compromise, especially this case, which you reported upstream too.
Moreover, I, at least, tend to simply ignore "always red" bot results.

**Comment by ColinFinck** [comment]
Given that the WINE developers are apparently on it (cf. https://bugs.winehq.org/show_bug.cgi?id=45719), I wouldn't mind merging this PR and reenabling the test only when it's fixed upstream.

But I leave it up to @learn-more and @ThFabba to give a final approval or denial. You can immediately merge this PR afterwards or close it.

**Pull request merged**